prime-router/src/main/resources/metadata/hl7_mapping/resources/Observation/OBX.yml

```diff
@@ -128,6 +128,24 @@ elements:
     resource: '%resource.extension(%`rsext-obx-observation`)'
     schema: classpath:/metadata/hl7_mapping/resources/Observation/OBXExtension.yml
 
+  # the specimen field in FHIR does not repeat so an extension is needed to capture additional specimen identifiers
+  # as such, use the extension when it is populated
+  - name: obx-related-specimen-identifier
```

Comment:
This change breaks _many_ of the FHIR to HL7 data tests because the FHIR bundles being tested have the `specimen` reference being resolved here, causing `OBX.33` to be populated when it previously was not. We may need to revisit whether this is needed.